discovery_engine_core/core/src/storage/sqlite.rs

```diff
@@ -341,6 +341,32 @@ impl Storage for SqliteStorage {
         Ok(())
     }
 
+    async fn clear_database(&self) -> Result<bool, Error> {
```

Comment:
:grey_question: just wondering re the return type, is it important to know whether there was at least 1 row deleted?